src/gatsby-node.ts

```diff
@@ -64,31 +68,41 @@ const createResolve: CreateResolve = ({
     return fileNodes[0]
   }
 
-  return fileNodes
+  // Return in original order
+  const fileNodesOrdered = filePaths.flatMap((filepath) => {
```

Comment:
Nice! But why `flatMap`  vs. `map`? I thought `find` returns the item itself & not wrapped in an array

Reply:
You're right. That context switch from Swift will getcha `flatMap / compactMap`. 

It shouldn't ever happen in this use case, but `find` can return `undefined`. Should remove any potential dirty values from the array.